app/views/cases/data_request_areas/show.html.slim

```diff
@@ -0,0 +1,39 @@
+- content_for :page_title do
+  = t('page_title.data_request_area_show_page')
+
+- content_for :heading
+  = t('.heading', data_request_area_type: @data_request_area.data_request_area_type.humanize)
+
+- content_for :sub_heading
+
+= link_to("Back", case_path(@case), class: 'govuk-back-link')
+
+- if @no_location_present
+  <br/>
+  <br/>
```

Comment:
It would be better to create this space using CSS.
Also, as this banner is used in a couple of places, it could be extracted into a partial.
(I don't mind this being done later as a separate PR)